autopush/endpoint.py

```diff
@@ -704,12 +703,11 @@ def put(self, router_type="", router_token="", uaid="", chid=""):
             return self._write_response(
                 400, 108, message="Invalid router")
         router = self.ap_settings.routers[router_type]
-        valid, router_token = router.check_token(router_token)
+        relo_url = self._relocate(router_type, uaid, chid)
+        valid, reply = router.check_token(router_token, self._write_response,
```

Comment:
In one spot its 'valid, writer', here its 'valid, reply'. I'm not sure why there's not consistency in the return naming.